proteus/config/__init__.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+if 'HOSTNAME' in os.environ:
```

Comment:
Now there is a `config` subpackage within `proteus`.  It has a default configuration that works on OS X, Linux, and Cygwin (maybe :) when HashDist is the build agent.  You can look through the `default.py` file and see that for the most part, so long as the system as a sane `prefix`, whether it was built by HashDist or no, you should be able to build Proteus.  However, there are going to be systems where things are different or weird enough that the logic shouldn't go into default.py, and that's the purpose of this configuration file.  For now, I suggest checking the system hostname, or some other flag available to the Python run-time for accessing this configuration file.  In this example, the `HOSTNAME` environment variable is set on both the front-end nodes (build-time) and the compute nodes (run-time), so it's a straightforward way to detect that Proteus is running on Garnet/Copper.